.github/workflows/ci.yml

```diff
@@ -40,7 +40,36 @@ jobs:
     secrets:
       githubToken: ${{ secrets.GITHUB_TOKEN }}
 
-  install:
+  setup-ubuntu-16:
```

Comment:
is `16` in the title a mistake? Seems like it should be labeled latest. Same for the OS entries below :)

Reply:
if its cause of the node version (which I expect it is), maybe a label of `setup-ubuntu-latest-node-16` is a little more descriptive (albeit longer)